packages/shared/test-suites/RecordingTestMetadata.ts

```diff
@@ -426,6 +409,9 @@ export async function processCypressTestRecording(
         annotations.forEach(annotation => {
           switch (annotation.message.event) {
             case "step:end": {
+              if (!testEndPoint || comparePoints(testEndPoint.point, annotation.point) < 0) {
+                testEndPoint = { point: annotation.point, time: annotation.time };
```

Comment:
Two questions:
1. are these annotations sorted? Would we need a Max function...
2. Are these annotations grouped per test case? I assume so...